### ISSUE TITLE
WKWebView in ASCellNode causes frame drop!

### ISSUE THREAD

**Issue by huang-kun** [TextureGroup/Texture#659]
There is a problem for using WKWebView in ASCellNode. I'm not sure if is a cache issue or any manual work to be setup for displaying web in cell, but when I scroll it with ASDK, the web seems to render every time it shows. Here is my demo link. Try it on iOS device (or simulator) and it hits the performance. 

The same cell works OK with UIWebView and UIKit.
https://github.com/huang-kun/WebBugDemo

**Comment by GeekTree0101** [comment]
try use viewBlock

```
_webNode = ASDisplayNode(viewBlock: {
            let webView = WKWebView(frame: frame)
            ....
            return webView
        })
```

**Comment by huang-kun** [comment]
@GeekTree0101  Have you actually tried on this view block on my demo project? It really makes no differences. Besides, calling -[ASDisplayNode initWithViewBlock:] is as same as calling [node.view addSubview:].

**Comment by GeekTree0101** [comment]
I guess, loadHTMLString is little heavy during scrolling feed, if you don't mind try to loadHTMLString on didEnterVisibleState or didEnterDisplayState, and Cell disappear from visible state then try to stopLoading()

<Intelligent preloading>
http://texturegroup.org/docs/intelligent-preloading.html

<UIWebView>
https://developer.apple.com/documentation/uikit/uiwebview/1617979-loadhtmlstring

**Comment by huang-kun** [comment]
I don't think that is helping. It still had bad scrolling after I removed the loadHTMLString method from cell node. It seems only scrolls OK without WKWebView in the cell node. If I replace it to UIWebview in cell node, the table freezes for a short period at initialization for loading HTML, but scrolling is perfectly fine. Maybe putting a web view in each table cell is kind of overuse in a real project, but I just want to point out the issue here. It's OK to end up without a better solution.

**Comment by SilverMagnolia** [comment]
@huang-kun have you resolved this issue? I have the same issue now. I think it's very critical issue because the reason why I use Texture is to improve scrolling performance. As you said, I tried to remove loadHTMLString method from ASCellNode with WKWebView and the scrolling issue still remains. is there a any solution for this? thanks.

**Comment by huang-kun** [comment]
@SilverMagnolia Sorry I can't fix the problem. I suggest you try to find an alternative solution instead of using WKWebView in ASCellNode.
